ui/components/app/signature-request/signature-request-header/signature-request-header.component.js

```diff
@@ -14,7 +14,14 @@ export default class SignatureRequestHeader extends PureComponent {
     return (
       <div className="signature-request-header">
         <div className="signature-request-header--account">
-          {fromAccount ? <AccountListItem account={fromAccount} /> : null}
+          {fromAccount ? (
```

Comment:
Can we add some snapshot tests for this condition?

Reply:
Added tests